tests/core/recipes/emt_recipes/test_emt_recipes.py

```diff
@@ -1,10 +1,14 @@
+from __future__ import annotations
```

Comment:
We should avoid `from __future__ import annotations` in the test suite.

Reply:
Yes, for some reason ruff --fix put them here automatically, each time I try to be careful to delete them manually, this one escaped!

Reply:
Ah, that's odd. I thought I had set up `pyproject.toml` to avoid that scenario (certainly, I don't want contributors to have to bother manually removing stuff!). I can double check.